src/content/docs/new-relic-solutions/get-started/intro-new-relic2.mdx

```diff
@@ -0,0 +1,195 @@
+  - /docs
+  - /docs/using-new-relic/cross-product-functions/install-configure/install-new-relic
+signupBanner:
+  text: Monitor and improve your entire stack. 100GB free. Forever.
+freshnessValidatedDate: never
+---
+
+import NROneMain from 'images/new-relic-one_screenshot-full_main.webp'
+
+import NRAlertsMain from 'images/alerts_screenshot-crop_main.webp'
+
+import NRApmMain from 'images/apm_screenshot-crop_main.webp'
+
+import NRBrowserMain from 'images/browser_screenshot-crop_main.webp'
+
+
+
+
+Eureka! That's the feeling you'll have when you learn how New Relic works. 
+With New Relic, you can monitor the performance of your applications, infrastructure, web browsers, and more. You can do this on a single platform with real-time data and analytics. You will identify any issues that may be impacting its functionality. It enables you to make informed decisions and optimize your systems for better performance.
+
+Let us help you and your team make your life better:
+
+  * Get full-stack observability across your enterprise to predict and detect issues before they impact customers.
+
+  * Put all your telemetry data, including open-source, on one secure scalable database with 700+ integrations.
+
+  * Visualize and analyze your entire stack in one unified experience, with AI-powered analytics to find and fix issues fast.
+
+  * Pay only for what you use.
+
+<img
+  title="New Relic main page"
+  alt="New Relic main page"
+  src={NROneMain}
+/>
+
+
+
+# Let's start today! [#start-today]
+
+<Steps>
+  <Step>
+    ## Sign up for New Relic
+
+    Are you ready?
+
+    <ButtonGroup>
+      <ButtonLink
+        role="button"
+        to="https://newrelic.com/signup"
+        variant="primary"
+      >
+        Sign up for New Relic
+      </ButtonLink>
+    </ButtonGroup>
+
+    Does your organization already have New Relic but you're just adding a new team or project? See [Add a new account](/docs/accounts/accounts-billing/account-structure/add-accounts).
+  </Step>
+  <Step>
+    ## Ingest some data
+
+    Follow our [guided install](https://one.newrelic.com/launcher/nr1-core.explorer?pane=eyJuZXJkbGV0SWQiOiJucjEtY29yZS5saXN0aW5nIn0=&cards[0]=eyJuZXJkbGV0SWQiOiJucjEtaW5zdGFsbC1uZXdyZWxpYy5ucjEtaW5zdGFsbC1uZXdyZWxpYyIsImFjdGl2ZUNvbXBvbmVudCI6IlZUU09FbnZpcm9ubWVudCIsInBhdGgiOiJndWlkZWQifQ==) to auto-discover your environment. The guided install walks you through a recommended setup. This step-by-step process makes the installation easier. It ensures that all needed parts are set up right.
+
+  </Step>
+
+  <Step>
+    ## Check out some dashboards
+
+    Explore your environment in real time with some out-of-the-box UI views. You'll see data in the **Infrastructure**, **APM**, and **Logs** UI pages. 
+
+    Learn how to [get started with dasboards](/docs/query-your-data/explore-query-data/dashboards/introduction-dashboards/).
+  </Step>
+
+  <Step>
+    ## Add more data
```

Comment:
Instead of sending people to the I/O site, I'd prefer to send people to the catalog in the UI.